packages/cpp/src/browser/cpp-protocol.ts

```diff
@@ -19,6 +19,7 @@ import { TextDocumentIdentifier, Command, MessageType } from '@theia/languages/l
 export interface ActionableMessage {
```

Comment:
This interface doesn't seem to be used.  Maybe we can just get rid of it?

Reply:
@simark I removed the unused interface as you suggested